ios/Podfile.lock

```diff
@@ -88,4 +106,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: c34e2287a9ccaa606aeceab922830efb9a6ff69a
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.7.4
```

Comment:
why the downgrade?

Reply:
that's my version of cocoapod in my system. This file is auto generated when u run `flutter packages get`.

Reply:
What version of flutter are you running? 

I think we should be on the beta channel for flutter atm.

Reply:
@ericjiang97 This is the Podfile.lock file, not Pubspec.yaml... Flutter version is not relevant :)